app/Http/Controllers/CommentController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pibbble\Http\Controllers;
+
+use Auth;
+use Response;
+use Pibbble\Comment;
+use Pibbble\Project;
+use Pibbble\Helpers\PostComment;
+use Pibbble\Http\Requests\CommentRequest;
+
+class CommentController extends Controller
+{
+    /**
+     * Saves project comment and returns JSON data
+     */
+    public function makeComment($id, CommentRequest $request, PostComment $comment)
+    {
+        $user = Auth::user();
+
+        return Response::json($comment->saveProjectComment($id, $user, $request));
```

Comment:
It means you don't need that line break. The times you actually need to use that `+` should be when you are concatenating the response data to the HTML elements.

Here you used it almost 100 times